app/assets/stylesheets/custom.scss

```diff
@@ -137,3 +137,20 @@ a {
 td {
   background-color: rgba(255, 255, 255, .5);
 }
+
+// variables, mixins
+@mixin box_sizing {
+  -moz-box-sizing:    border-box;
+  -webkit-box-sizing: border-box;
+  box-sizing:         border-box;
+}
+
+/* miscellaneous */
+
+.debug_dump {
+  clear: both;
```

Comment:
Rule sets should be ordered as follows: @extends, @includes without @content, properties, @includes with @content, nested rule sets